k8s-two-tier/web-deployment.yml

```
@@ -0,0 +1,94 @@
+      containers:
+      - name: tracker-pod
+        image: swayamnakshane/expenseapp
+        ports:
+          - containerPort: 8080
+        env:
+        - name: SPRING_DATASOURCE_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: mysql-secret
+              key: SPRING_DATASOURCE_PASSWORD
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)